src/scene-entry-manager.js

```diff
@@ -268,6 +240,84 @@ export default class SceneEntryManager {
         }
       }
     });
+
+    let currentVideoShareEntity;
+    let isHandlingVideoShare = false;
```

Comment:
Is `isHandlingVideoShare` ever false when `currentVideoShareEntity` is defined? If not I would ditch the extra bool.